environment.yml

```diff
@@ -13,3 +13,4 @@ dependencies:
     - prompt_toolkit==2.0.9
     - azure-cli==2.0.75
     - azureml-sdk[notebooks]==1.0.69
+    - git+https://github.com/Microsoft/StatisticsTracker
```

Comment:
Instrumentation should not be in the published path. This would be fine for the instrument branch.